packages/shared/lib/sdk/sync.ts

```diff
@@ -417,9 +412,24 @@ export class NangoAction {
         return (metadata['fieldMapping'] as Metadata) || {};
     }
 
-    public async log(content: string, userDefinedLevel?: UserLogParameters): Promise<void> {
+    /**
+     * Log
+     * @desc Log a message to the activity log which shows up in the Nango Dashboard
+     * note that the last argument can be an object with a level property to specify the log level
+     * example: await nango.log('This is a log message', { level: 'error' })
+     */
+    public async log(...args: any[]): Promise<void> {
+        const lastArg = args[args.length - 1];
+        const userDefinedLevel = typeof lastArg === 'object' && lastArg.level ? lastArg : undefined;
```

Comment:
you can write a function with a type guard. something like this:
```
const isUserDefinedLevel(object: any): object is userDefinedLevel {
    return 'level' in object;
}
 const userDefinedLevel = isUserDefinedLevel(lastArg)? lastArg : undefined;
```

Reply:
Thanks, added with https://github.com/NangoHQ/nango/pull/1601/commits/ceb892eaaedb1c680a2f2332cd9325fa1999b60d#diff-f52f0d61cd6802499f5a31aca89d3aec3df278a751065390c02b6bbdb24c0840R433